config.go

```diff
@@ -256,7 +256,7 @@ type Config struct {
 	RawRESTListeners  []string `long:"restlisten" description:"Add an interface/port/socket to listen for REST connections"`
 	RawListeners      []string `long:"listen" description:"Add an interface/port to listen for peer connections"`
 	RawExternalIPs    []string `long:"externalip" description:"Add an ip:port to the list of local addresses we claim to listen on to peers. If a port is not specified, the default (9735) will be used regardless of other parameters"`
-	ExternalHosts     []string `long:"externalhosts" description:"A set of hosts that should be periodically resolved to announce IPs for"`
+	ExternalHosts     []string `long:"externalhosts" description:"Add a hostname:port that should be periodically resolved to announce IPs for. If a port is not specified, the default (9735) will be used."`
```

Comment:
Should we also add "Can be specified multiple times" to make it clear it's a list of hosts? In the code you see all the flags that are slices (`[]string`) that can be specified multiple times. But when just looking at `lnd --help` it's not really clear.

Reply:
That's fine with me. I just followed the model of the options in the lines above like `externalip` which doesn't say "can be specified multiple times". I think that they say they "add" an option implies that they can be specified multiple times but I also see how it might not be that clear.

Should they all mention explicitly that they can be specified multiple times?

Reply:
Yeah, you're right, best stick to how the other multiple options are worded. We can overhaul the whole config description in another PR.